Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -946,6 +946,10 @@
       damage: 8
     - type: Item
       size: Small
+      shape:
+      - 1,0,1,0
+      - 1,1,2,1
+      - 0,2,0,2
```

Comment:
diagonal connections look really bad. i maybe try a shape like this?
```
XXX
.X.
```

or 

```
XXX
XXX
.X.
.X.
```

also, if the size is small, the total area needs to match that for consistency purposes. Personally, i don't really see the need to make it any bigger than 2x1. The joke isn't really that funny.